pkg/assetfetcher/config.go

```diff
@@ -0,0 +1,35 @@
+package assetfetcher
+
+import "context"
+
+// AssetFetcherConfig holds configuration to connect to Nexpose
+// and make a call to the fetch assets API
+type AssetFetcherConfig struct {
+	Host                 string
+	Username             string
+	Password             string
+	AssetFetcherPageSize int
+}
+
+// Name is used by the settings library and will add a "NEXPOSE_"
+// prefix to AssetFetcherConfig environment variables
+func (c *AssetFetcherConfig) Name() string {
+	return "Nexpose"
+}
+
+// AssetFetcherConfigComponent satisfies the settings library Component
+// API, and may be used by the settings.NewComponent function.
+type AssetFetcherConfigComponent struct{}
+
+// Settings can be used to populate default values if there are any
+func (*AssetFetcherConfigComponent) Settings() *AssetFetcherConfig { return &AssetFetcherConfig{} }
```

Comment:
Question: Is there a reasonable default for page size we could return here?

Reply:
There is. I wasn't sure how this part would work if there's only one variable that has a default and the others don't. Would I just set that one variable here and not the others?

Reply:
> Would I just set that one variable here and not the others?
Yeah, you can set only some of them and only those will have default values.

Reply:
Sweet, I'll add it.